doc/source/whatsnew/v1.1.0.rst

```diff
@@ -315,6 +315,7 @@ Groupby/resample/rolling
 
 - Bug in :meth:`GroupBy.apply` raises ``ValueError`` when the ``by`` axis is not sorted and has duplicates and the applied ``func`` does not mutate passed in objects (:issue:`30667`)
 - Bug in :meth:`DataFrameGroupby.transform` produces incorrect result with transformation functions (:issue:`30918`)
+- Bug in :meth:`pandas.core.groupby.RollingGroupby.apply` was including the grouped-by column in its values, rather than just in the index (:issue:`32262`)
```

Comment:
this is going to need a separete section in api breaking changes showing the before & after. this is a big user facing change.